fmp_server.py

```diff
@@ -26,9 +26,61 @@
 from other.utils import hexdump, server_base, server_main
 
 
+def construct_broadcast(text="Hello, World!"):
+    assert len(text) > 0
+    MAX_CHAR_LENGTH = 30
+    broadcast_segments = []
+    words = text.split(" ")
+
+    mod_words = []
+
+    for word in words:
+        curr_word = word
+        while len(curr_word) > 30:
+            mod_words.append(curr_word[0:30])
+            curr_word = curr_word[30:]
+        if len(curr_word) > 0:
+            mod_words.append(curr_word)
+
+    curr_broadcast = mod_words[0]
+    for word in mod_words[1:]:
+        if len(curr_broadcast) + 1 + len(word) <= 30:
+            curr_broadcast += " "
+            curr_broadcast += word
+        else:
+            broadcast_segments.append(curr_broadcast)
+            curr_broadcast = word
+    if len(curr_broadcast) > 0:
+        broadcast_segments.append(curr_broadcast)
+    
+    broadcasts = []
+    for broadcast_text in broadcast_segments:
+        data = struct.pack(">B", 0x01)
+        info = pati.MessageInfo()
+        info.text_color = pati.Long(0x00fffff0)
+        info.sender_id = pati.String("000000")
+        info.sender_name = pati.String("[MH3SP]")
+        data += info.pack()
+        data += pati.lp2_string(broadcast_text)
+        print("BROADCAST: " + broadcast_text)
```

Comment:
Can't we log this message properly, somewhere, rather than using raw prints?

Reply:
By the time it gets to the FmpServer, it's already bundled up into a binary. This function doesn't have access to any loggers since it's not local to any of the `PatServer` classes.

Reply:
Is this print necessary since the server could log any packets including broadcast ones?